timescaledb/getting-started/create-cagg/create-cagg-basics.md

```diff
@@ -0,0 +1,132 @@
+
+```sql
+SELECT
+	time_bucket('1 day', "time") AS day,
+	symbol,
+	max(price) AS high,
+	first(price, time) AS open,
+	last(price, time) AS close,
+	min(price) AS low
+FROM stocks_real_time srt
+GROUP BY day, symbol
+ORDER BY day DESC, symbol;
+```
+
+## Step 2: Create continuous aggregate from aggregate query
+
+Now that you have the aggregation query, you can use it to create a continuous aggregate. 
+
+The `CREATE MATERIALIZED VIEW` command triggers the database to create a materialized view with 
+the given name, in this case `stock_candlestick_daily`. In the next line, 
+`WITH (timescaledb.continuous)` lets the database know that you want to create a continuous 
+aggregate and not just a generic materialized view. Then, the `AS` keyword is needed to specify 
+the aggregate query you want to use for creating the continuous aggregate. 
+
+To do this, use the following code:
+
+```sql
+CREATE MATERIALIZED VIEW stock_candlestick_daily
+WITH (timescaledb.continuous) AS
+SELECT
+	time_bucket('1 day', "time") AS day,
+	symbol,
+	max(price) AS high,
+	first(price, time) AS open,
+	last(price, time) AS close,
+	min(price) AS low
+FROM stocks_real_time srt
+GROUP BY day, symbol;
+```
+
+Notice that the `SELECT` statement is the same query you wrote earlier without the `ORDER BY` clause.
+By default, this code both creates the aggregate *and* materializes the aggregated data.
+That means the view is created *and* populated with the aggregate calculations from
+your existing hypertable data. 
+
+  <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/continuous-aggregate.jpg" alt="Continuous aggregate upon creation"/>
+
+The query may take some time to run because it needs to perform these calculations. 
+After the calculation results are stored, querying the data from the 
+continuous aggregate is much faster. Run the following query to get all the data in your 
+continuous aggregate, and note how much faster this is than running the aggregate `SELECT` 
+query on the raw hypertable data.
+
+```sql
+SELECT * FROM stock_candlestick_daily
+ORDER BY day DESC, symbol;
+```
+
+By default, when you query a continuous aggregate, TimescaleDB also includes recent
+data that hasn't yet been materialized. In the diagram above, that corresponds to the last
+three points of raw data, which belong to an incomplete bucket. This unmaterialized data
+is aggregated in real time to provide the most up-to-date results for your query.
+
+<highlight type="note">
+To inspect details about a continuous aggregate, such as its 
+configuration or the query used to define it, use the following 
+informational view:
+
+```sql
+SELECT * FROM timescaledb_information.continuous_aggregates;
+```
+Results:
+
+|hypertable_schema|hypertable_name |view_schema|view_name              |view_owner|materialized_only|compression_enabled|materialization_hypertable_schema|materialization_hypertable_name|view_definition |
```

Comment:
Table formatting seems to be broken within callouts:
<img width="706" alt="image" src="https://user-images.githubusercontent.com/26616127/163655230-505c8007-65a2-496e-92ae-7349cc93e9b0.png">